### PR TITLE
Add data: Sudan Al Gadaref CEO 2020 

### PR DESCRIPTION
**Pull request by MsPixels**

### CODE REVIEW

File: data/report.txt [hunk 1]
```diff
@@ -386,3 +386,12 @@ eo_data_complete    1350
 ✔ training amount: 405, positive class: 4.9%
 ✔ validation amount: 458, positive class: 5.9%
 ✔ testing amount: 487, positive class: 2.3%
+
+
+
+SudanAlGadarefCEO2020 (Timesteps: 24)
```
Comment (bhyeh) [inline data/report.txt:392]: Looks like EO data is completed!

**Review by bhyeh — APPROVED** [review]

File: data/report.txt [hunk 2]
```diff
@@ -386,3 +386,12 @@ eo_data_complete    1350
...
+----------------------------------------------------------------------------
+eo_data_complete    1500
+✔ training amount: 449, positive class: 42.1%
+✔ validation amount: 518, positive class: 37.1%
+✔ testing amount: 533, positive class: 37.7%
```
Comment (bhyeh) [inline data/report.txt:397]: looks like EO data is completed exporting!

**Comment by MsPixels** [comment]
@bhyeh and @adebowaledaniel, kindly review this when you have the chance. Thanks!

**Review by adebowaledaniel — APPROVED** [review]
LGTM

**Pull request merged**